ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/product/AboutSection.java

```diff
@@ -147,6 +147,7 @@ private void handleBrowse() {
 		FileExtensionsFilter filter = new FileExtensionsFilter();
 		filter.addFileExtension("gif"); //$NON-NLS-1$
 		filter.addFileExtension("png"); //$NON-NLS-1$
+		filter.addFileExtension("svg"); //$NON-NLS-1$
```

Comment:
I'm not sure if SVG can be generally allowed for `images that are associated with the application window`? This might be renderer natively by the OS.